src/db/spanner/models.rs

```diff
@@ -647,12 +650,21 @@ impl SpannerDb {
         }
     }
 
+    fn erect_tombstone(&self, user_id: u32) -> Result<()> {
+        self.sql("INSERT INTO user_collections (userid, collection, last_modified) values (@userid, \"0\", @modified)")?
```

Comment:
We need an UPSERT here like the mysql version. Spanner only has one in its mutation API.

Feel free to just issue a DELETE of the tombstone beforehand for now (way easier than  writing select + if exists insert else update)

I have some uncommitted code for using Mutations that we can change this to later